projects/novo-elements/src/elements/picker/extras/mixed-multi-picker-results/MixedMultiPickerResults.ts

```diff
@@ -17,6 +17,7 @@ export interface IMixedMultiPickerOption {
         filterValue?: any,
     }[];
     getSecondaryOptionsAsync?(): Promise<{value: string, label: string}[]>;
+    clearSecondaryOptions?: BehaviorSubject<Boolean>;
```

Comment:
Why is `clearSecondaryOptions` a BehaviorSubject? I'm curious about how this is used by the component, it might also help to update the examples in novo-elements for the MuxesMultiPickerResults component.